templates/server-ingress.yaml

```
@@ -2,6 +2,9 @@
 {{- if ne .mode "external" }}
 {{- if .Values.server.ingress.enabled -}}
 {{- $serviceName := include "vault.fullname" . -}}
+{{- if and (eq .mode "ha" ) (and (eq (.Values.server.service.enabled | toString) "true" ) (eq (.Values.global.enabled | toString) "true")) }}
+{{- $serviceName := printf "%s-%s" $serviceName "active" -}}
```

I think this needs to be `=` instead of `:=`, because serviceName is already defined above.
```suggestion
{{- $serviceName = printf "%s-%s" $serviceName "active" -}}
```

---

you're right. I originally did that but then had a problem with it showing `unexpected "=" in operand`. However, that's because I had switched to an older version of helm which appears to have a bug. I will revert this to my original version without the :